external/Directory.Build.props

```diff
@@ -1,10 +1,10 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003"> 
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <IntermediateOutputPath>$(IntermediateOutputPath)$(OSGroup)-$(ArchGroup)/</IntermediateOutputPath>
     <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
     <ContainsPackageReferences>true</ContainsPackageReferences>
     <!-- We need configuration-specific assets files. -->
+    <IntermediateOutputPath>$(IntermediateOutputPath)$(OSGroup)-$(ArchGroup)/</IntermediateOutputPath>
```

Comment:
Is this change necessary?

Reply:
There is no change here, I only move the line down?

Reply:
Right - why?

In general, moving things around for no reason makes it harder on reviewers, but even more importantly it makes it harder for people to inspect source history.

Was the idea that setting `IntermediateOutputPath` and `RestoreOutputPath` should always go together?

Reply:
> Was the idea that setting IntermediateOutputPath and RestoreOutputPath should always go together?

Correct. Setting IntermediateOutputPath should always following with setting the RestoreOutputPath, that's why I put them together.